doc/frameworks/symfony.md

```diff
@@ -0,0 +1,481 @@
+            - { name: doctrine_mongodb.odm.event_listener, event: 'preRemove' }
+        calls:
+            # Uncomment the below call if using attributes, and comment the call for the annotation reader
+            # - [ setAnnotationReader, [ '@gedmo.mapping.driver.attribute' ] ]
+            # The `annotation_reader` service was deprecated in Symfony 6.4 and removed in Symfony 7.0
+            - [ setAnnotationReader, [ '@annotation_reader' ] ]
+
+    # Gedmo References Extension Listener
+    gedmo.listener.references:
+        class: Gedmo\References\ReferencesListener
+        tags:
+            - { name: doctrine_mongodb.odm.event_listener, event: 'postLoad' }
+            - { name: doctrine_mongodb.odm.event_listener, event: 'loadClassMetadata' }
+            - { name: doctrine_mongodb.odm.event_listener, event: 'prePersist' }
+            - { name: doctrine_mongodb.odm.event_listener, event: 'preUpdate' }
+        calls:
+            # Uncomment the below call if using attributes, and comment the call for the annotation reader
+            # - [ setAnnotationReader, [ '@gedmo.mapping.driver.attribute' ] ]
+            # The `annotation_reader` service was deprecated in Symfony 6.4 and removed in Symfony 7.0
+            - [ setAnnotationReader, [ '@annotation_reader' ] ]
+```
+
+### Extensions Compatible with ORM Only
+
+> [!NOTE]
+> This example shows the configuration when using the ORM and `DoctrineBundle` with a single default entity manager. When using an application with multiple managers, a separate tag is needed with the `connection` attribute for each connection.
+
+```yaml
+services:
+    # Gedmo Uploadable Extension Listener
+    gedmo.listener.uploadable:
+        class: Gedmo\Uploadable\UploadableListener
+        tags:
+            - { name: doctrine.event_listener, event: 'loadClassMetadata'}
+            - { name: doctrine.event_listener, event: 'preFlush'}
+            - { name: doctrine.event_listener, event: 'onFlush'}
+            - { name: doctrine.event_listener, event: 'postFlush'}
+        calls:
+            # Uncomment the below call if using attributes, and comment the call for the annotation reader
+            # - [ setAnnotationReader, [ '@gedmo.mapping.driver.attribute' ] ]
+            # The `annotation_reader` service was deprecated in Symfony 6.4 and removed in Symfony 7.0
+            - [ setAnnotationReader, [ '@annotation_reader' ] ]
+```
+
+## Registering Mapping Configuration
+
+When using the [Loggable](../loggable.md), [Translatable](../translatable.md), or [Tree](../tree.md) extensions, you will
+need to register the mappings for these extensions to your object managers.
+
+> [!NOTE]
+> These extensions only provide mappings through annotations or attributes; if using annotations, you will need to ensure the deprecated [`doctrine/annotations`](https://www.doctrine-project.org/projects/annotations.html) library is installed and enabled.
```

Comment:
AFAIK, the deprecation mentioned here operates on the library implementation, [not on the library itself](https://github.com/doctrine/annotations/blob/94f40ad7ecbc6931958faa8a57c48dce5da2b468/README.md?plain=1#L1-L5).

>⚠️ PHP 8 introduced [attributes](https://www.php.net/manual/en/language.attributes.overview.php), which are a native replacement for annotations. As such, this library is considered feature complete, and should receive exclusively bugfixes and security fixes.

Reply:
I understand.
IMO, from our perspective, the important thing in our documentation is our deprecation for the annotation mapping driver, regardless of the library deprecation.
https://github.com/doctrine-extensions/DoctrineExtensions/blob/e85560ed96f977b8c29428a99222cb2ef2f0e80d/README.md?plain=1#L64-L65

Reply:
OK, last commit should (hopefully) make this a bit more clear across all pages I've updated so far.

Reply:
Thank you!